proposals/bip-002.md

```diff
@@ -100,7 +100,9 @@ tests.
 - The integration-test implementation would have to be modified to provide the
```

Comment:
This is just a use case, not sure that we need to specify how this annotation status will be used, how we implement this report of if we do it should be decided without having to alter this spec if we decide to do it differently. It may be interesting to add it as a use case in the rationale, though.